### ISSUE TITLE
I cannot see existing saved accounts into Splice

### ISSUE THREAD

**Issue by HerminaC** [oyiptong/splice#261]
http://splice_tiles.dev.mozaws.net/ -> Campaign App -> Account - undefined.
When I try to select an already existing account in order to add a new campaign/adgroup/tile to it, I don't see the existing accounts anymore. 
If I choose the option to create a tile directly, I can upload my image, name it and use all functionalities for creating a tile but I cannot save it as there is no account to link it to.  Please see attach. 

![no saved accounts](https://cloud.githubusercontent.com/assets/7406447/11396708/89aba51c-936b-11e5-9a0d-1c9115614fcf.png)
![no saved acc - tiles uploaded](https://cloud.githubusercontent.com/assets/7406447/11396717/8edfaf74-936b-11e5-9dfd-5ff7a11cebfd.png)

**Comment by k88hudson** [comment]
Same here, I think this is an issue with the dev server crashing

**Comment by k88hudson** [comment]
Ok, so this is *not* related to the crash this morning I don't think;

I was able to reproduce this with the following steps:

1. Visit home (http://splice_tiles.dev.mozaws.net/)
2. List of recent accounts has some with `Account - undefined`
3. Click on one of the `Account - undefined` links
4. Goes to http://splice_tiles.dev.mozaws.net/campaign/#/accounts/3, account does have a name
5. Go back to home (http://splice_tiles.dev.mozaws.net/)
6. Account name is now fixed/no longer has `undefined`  in the recent links
